src/App.tsx

```diff
@@ -1,19 +1,23 @@
 import React, { FC } from 'react';
 import { BrowserRouter } from 'react-router-dom';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
 import Header from './components/Header';
 import I18nProvider from './providers/I18n';
 import AuthProvider from './providers/Auth';
 import AppRoutes from './AppRoutes';
 
 const App: FC = () => (
-  <I18nProvider>
-    <BrowserRouter>
-      <AuthProvider>
-        <Header />
-        <AppRoutes />
-      </AuthProvider>
-    </BrowserRouter>
-  </I18nProvider>
+  <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
It's just a comment

I did not see to provider for picker wrap all applications, but maybe it is normal

Reply:
To me it is ok. It has been implemented according to the setup guide.